alert_generator/server.go

```diff
@@ -84,6 +84,13 @@ func newAlertsServer(port string, disabled bool, logger log.Logger, messageParse
 }
 
 func (as *alertsServer) ServeHTTP(res http.ResponseWriter, req *http.Request) {
+	// some proxies/sinks like webhook.site might use a preflight check
```

Comment:
(A habit/practice from prometheus/prometheus)
```suggestion
	// Some proxies/sinks like webhook.site might use a preflight check.
```